### PR TITLE
Remove plugins we don't use on Admin.

### PR DESCRIPTION
**Pull request by feanil**
These are a few plugins that I don't think we use on admin anymore and should be removed from this list.

@edx/devops

### CODE REVIEW

File: playbooks/roles/jenkins_admin/defaults/main.yml [hunk 1]
```diff
@@ -105,7 +103,6 @@ jenkins_admin_plugins:
   - { name: "parameterized-trigger", version: "2.25" }
   - { name: "multiple-scms", version: "0.3" }
   - { name: "git", version: "2.2.2" }
```
Comment (maxrothman) [inline playbooks/roles/jenkins_admin/defaults/main.yml:105]: We don't use the git plugin anymore?

Comment (feanil) [inline playbooks/roles/jenkins_admin/defaults/main.yml:105]: You're reading the diff incorrectly.

Comment (maxrothman) [inline playbooks/roles/jenkins_admin/defaults/main.yml:105]: (doh)

**Comment by maxrothman** [comment]
:+1:

**Pull request merged**